app/pricecheck.js

```diff
@@ -1,37 +1,50 @@
 var itemName = document.title.split("Listings for ")[1];
-
 var LoungeUser = new User();
+
 chrome.storage.local.get(['currencyConversionRates'], function(result) {
+
+    var GetCSGLSchemaUrl = 'http://csgolounge.com/api/schema.php';
```

Comment:
The Upper case format should indicate it's not. However; there's an even better way to distinguish the var, apparently[1] all caps snake case is the proper way to go.

E.g. CSGL_SCHEMA_ENDPOINT or CSGL_SCHEMA_URL

[1] - http://stackoverflow.com/questions/130396/are-there-constants-in-javascript
Soon - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const